pom.xml

```diff
@@ -1529,6 +1529,13 @@
       <id>sunny</id>
       <properties>
         <failsafe.groups>SunnyDay</failsafe.groups>
+        <!--
+          exclude junit-platform-suite engine from this list of engines to run,
+          so it skips the SimpleSuite tests, which contain no sunny tests, to
+          prevent the suite engine from running the BeforeSuite and AfterSuite
+          minicluster setup and teardown unnecessarily for zero tests
+        -->
+        <failsafe.includeJUnit5Engines>junit-jupiter</failsafe.includeJUnit5Engines>
```

Comment:
Won't this be a problem if a SimpleSuite test is added to sunny in the future? Maybe we are saying a SimpleSuite test can not be a sunny test (which is fine as including a test as SimpleSuite is more of thing that's nice to include when we can). If this is the case, this should be added to the description for the `SimpleSharedMacTestSuiteIT` class

Reply:
Yes it would be a problem if we did that. But, if we did that, we can just remove this line. This exists only to avoid starting and stopping a minicluster for zero tests. If we add a sunny test to the SimpleSuite, then we wouldn't have zero tests, and then it would be okay to have it started. That's why I put this comment here.